nextcloud/android
# Show unread comments count

Shows unread comments as "talk bubble".

![2018-12-05-102004](https://user-images.githubusercontent.com/5836855/49503282-5a06ec80-f877-11e8-8d50-d0aa8ddd6bf3.png)

To test
- share a file/folder with your user from user B
- as user B: comment on file/folder
- see "bubble" after refresh in app
- click on comment -> see comment
- back: bubble is removed
